app/controllers/reports_controller.rb

```diff
@@ -21,6 +21,8 @@ def active
   def history
     reports = ReportFilter.new(params).query
     @reports = params[:show_all] ? reports : reports.page(params[:page])
+    @search_item = params[:search_by_keyword]
+    @report_count = @reports.length
```

Comment:
I think this variable is a bit redundant since `@reports` are already being passed in. You could instead use `.length` in the view.